web/gui/dashboard_info.js

```diff
@@ -4201,7 +4201,35 @@ netdataDashboard.context = {
     },
 
     // ------------------------------------------------------------------------
-    // WMI
+    // WMI (Process)
+
+    'wmi.processes_cpu_time': {
+        info: 'Total CPU utilization. The amount of time spent by the process in <a href="https://en.wikipedia.org/wiki/CPU_modes#Mode_types" target="_blank">user and privileged</a> modes.'
+    },
+    'wmi.processes_handles': {
+        info: 'Total number of handles the process has open. This number is the sum of the handles currently open by each thread in the process.'
```

Comment:
Can you explain what handles are?

Reply:
I will add a link for this from MSDN official documentation, but I am not sure if we can add few words to explain Microsoft handles.